src/app/app-routing.module.ts

```diff
@@ -1,10 +0,0 @@
-import {NgModule} from '@angular/core';
-import {Routes, RouterModule} from '@angular/router';
-
-const routes: Routes = [];
-
-@NgModule({
-  imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule],
-})
-export class AppRoutingModule {}
```

Comment:
come mai questo file è stato eliminato? andrebbe rimosso anche la dipendenza da angular routing se decidiamo di non usarlo. se non sbaglio serviva a fare pagine multiple invece che tenere tutto in una single-page application

Reply:
In tutta onestà non lo so, quando ho iniziato a lavorarci non c'era già però tutti i moduli necessari erano importati nel file app-module.ts e a livello di funzionalità non cambia niente

Reply:
Magari decidiamo se toglierlo del tutto o no? Io lo terrei e farei 2-3 pagine diverse. Tipo:

1. Landing page con login se sei non auth e lista organizzazioni se sei auth
2. Pagina di gestione organizzazione (sia dettagli che places)
3. Pagina dei report (accessibile da gestione org)

E Tutto il resto della funzionalità si implementa internamente a queste pagine

Reply:
L'ho eliminato perché tenendo le impostazioni di routing in _src/app/core/app-routing.module.ts_ non funzionavano i redirect. Ovviamente non ho eliminato interamente il routing, l'ho semplicamente spostato un livelo più in alto piuttosto di perdere troppo tempo a farlo funzionare in una sottocartella. Ora si trova in _src/app/app.module.ts_

Reply:
segniamo la conversazione come resolved e aggiungo un nuovo issue, sta roba va sistemata ma magari più avanti